pandas/core/dtypes/cast.py

```diff
@@ -1535,7 +1539,7 @@ def maybe_cast_to_datetime(value, dtype: DtypeObj, errors: str = "raise"):
     return value
 
 
-def find_common_type(types: List[DtypeObj]) -> DtypeObj:
+def find_common_type(types: Union[List[DtypeObj], Series]) -> DtypeObj:
```

Comment:
where are we passing a Series? we shouldn't do this

Reply:
in `pandas/tests/arrays/sparse/test_accessor.py::test_to_coo`

Reply:
I'll dig deeper at the weekend

Reply:
@jreback the problem is that when we do
```
        df = pd.DataFrame({"A": [0, 1, 0], "B": [1, 0, 0]}, dtype="Sparse[int64, 0]")
        result = df.sparse.to_coo()
```
then, in `to_coo` from pandas/core/arrays/sparse/accessor.py, we have
```
        dtype = find_common_type(self._parent.dtypes)
```
and so this is how we pass a Series (`self._parent.dtypes` is a Series).

Is there a better way to fix this than
```
        dtype = find_common_type(self._parent.dtypes.to_list())
```
?